webf/lib/src/module/local_storage.dart

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2022-present The WebF authors. All rights reserved.
+ */
+
+import 'dart:async';
+import 'dart:convert';
+import 'package:path/path.dart' as path;
+import 'package:hive/hive.dart';
+import 'package:webf/foundation.dart';
+import 'package:webf/module.dart';
+
+class LocalStorageModule extends BaseModule {
+  @override
+  String get name => 'LocalStorage';
+
+  static String getBoxKey(ModuleManager moduleManager) {
+    String url = moduleManager.controller.url;
+    final bytes = utf8.encode(url);
```

Comment:
Should use origin as the key, not the whole URL.
Spec: https://storage.spec.whatwg.org/#obtain-a-storage-key-for-non-storage-purposes

Reply:
done